src/oc/notify/resources/notification.clj

```diff
@@ -162,7 +169,8 @@
     :content comment-body
     :mention? false
     :reminder? false
-    :author author}))
+    :author author
+    :url-path (str "/" org-id "/" board-id "/post/" entry-id "/comment/" interaction-id)}))
```

Comment:
@cjsauer maybe add a function to create this URL? To keep it in one place only, also you can try using `clojure.string/join`, not sure if it's better though but seems cleaner to me.